### PR TITLE
Truth data README hospitalisation clarification

### PR DESCRIPTION
**Pull request by kathsherratt**
Update truth data README for hospitalisations (markdown preview [here](https://github.com/epiforecasts/covid19-forecast-hub-europe/blob/2936d3031ee31966add1323fe335ee5603757adc/data-truth/README.md))
- Clarify exactly which countries are included with hospitalisation truth data
- Plot only those countries in the README

There remains some problem with the JHU part of this Rmd, to be fixed separately

### CODE REVIEW

File: data-truth/README.md [hunk 1]
```diff
@@ -41,15 +41,31 @@ We gather general hospital admissions data from various sources. See
 separate [Hospitalisations
 README](https://github.com/epiforecasts/covid19-forecast-hub-europe/tree/main/code/auto_download/hospitalisations#readme).
 
+Hospitalisation data can be difficult to produce and interpret, and is
+not consistent across all the countries in the ECDC Forecast Hub. To
+keep data and forecasts consistent, we include hospitalisations
+forecasts for the following locations only:
+
+Czechia, Iceland, Ireland, Italy, Latvia, Liechtenstein, Malta, Belgium,
```
Comment (sbfnk) [inline data-truth/README.md:49]: Maybe pull the list of countries from the file and sort automatically (can get the countries from the repo and then `paste(countries, collapse = ", ")` I think

**Review by sbfnk — APPROVED** [review]
Looks good. I wonder if we should store the `jpg` in an azure blob (@Bisaloo?) but probably not a priority for now.

File: data-truth/README.md (hunk 1 above)
Comment (kathsherratt) [reply data-truth/README.md:49]: Yep - this is done in the Rmd [here](https://github.com/epiforecasts/covid19-forecast-hub-europe/pull/1203/files?authenticity_token=ovxJ4SQQ2Axdcr8jJEwA2Jop7LqC2QIu%2BowCKOhdnaDMktmZVY9ujmAvz9eaUZB9F3XmsJWfUBAR3Xr4%2B8TqfA%3D%3D&file-filters%5B%5D=.Rmd&file-filters%5B%5D=.jpg#diff-d8ff620e0cb21b8eedaa87fcc45d45b012edfec22824ffb5336ef3674771d300R81) which regenerates the readme every day.

Comment (sbfnk) [reply data-truth/README.md:49]: Ah yes, my bad, sorry! Really minor but maybe use `sort(hosp_locations)` there?

Comment (kathsherratt) [reply data-truth/README.md:49]: Ah sorry will do, I'll just push to main since I was a bit quick to merge.

**Pull request merged**